config/const.py

```diff
@@ -1,13 +1,17 @@
 from os import sep
 from os.path import join, dirname, realpath
+import datetime
 
 PROJECT_ROOT = join(sep, *dirname(realpath(__file__)).split(sep)[: -1])
-print(PROJECT_ROOT)
-TIMESTAMP = "%Y-%m-%d_%H-%M-%S"
+
+TIMESTAMP = str(datetime.datetime.now())
```

Comment:
Is this correct? In the previous version, TIMESTAMP means time-format. Now it means the loaded time. In addition, I don't think we should use the current TIMESTAMP globally. It doesn't show the correct timestamp.